src/Plugin/Styleguide/defaultStyleguide.php

```diff
@@ -759,8 +759,7 @@ public function items() {
     );
 
     // Invoke hook_styleguide_alter()
-    $items = \Drupal::service('module_handler')->invokeAll('styleguide_alter', array($items));
-    //$items->merge($new_items);
+    \Drupal::service('module_handler')->alter('styleguide', $items);
```

Comment:
It's already added to the class as dependency injection. So better to use $this->moduleHandler